controllers/util/secrets_util.go

```diff
@@ -230,7 +281,28 @@ func newS3ConfigurationSecret(s3SecretRef corev1.SecretReference, targetns strin
 	}
 }
 
-func newConfigurationPolicy(name string, object runtime.RawExtension) *cpcv1.ConfigurationPolicy {
+func newVeleroSecret(s3SecretRef corev1.SecretReference, fromNS, veleroNS, keyName string) *localSecret {
+	return &localSecret{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Secret",
+			APIVersion: "v1",
+		},
```

Comment:
Why not use `corev1.Secret{'? than we don't need to specify the TypeMeta.

I guess you keep the existing style to minimize the change?